.github/workflows/build_workflow.yml

```diff
@@ -37,7 +37,7 @@ jobs:
 
       - if: ${{ steps.skip_check.outputs.should_skip != 'true' }}
         id: file_changes
-        uses: trilom/file-changes-action@v1.2.3
+        uses: trilom/file-changes-action@1.2.4
```

Comment:
Interesting!  I think the tag doesn't include the `v` so the bot is correctly removing it.  It's interesting that it worked before.